package.json

```diff
@@ -113,6 +113,10 @@
           "value": "zh-tw",
           "description": "繁體中文 (zh-tw)"
         },
+        {
+          "value": "cz",
```

Comment:
Considering alphabetic order,
please move these addition right after `"value": "be",`

Reply:
Ok, I'll do it

Reply:
And Czech is čeština in Czech.

Reply:
@liuderchi I suppose I need to create another pull request for that, am I right?